packages/language/src/validation/lsp-codes.ts

```diff
@@ -19,6 +19,12 @@ export const LspCodes = {
       message:
         "Could not resolve include directive. Plugin configuration is missing",
     },
+    FileNotFound: {
+      code: "LSPIR002",
+      severity: Severity.S,
+      message: (includefilename: string) =>
+        `INCLUDE file "${includefilename}" not found.`,
+    },
```

Comment:
Question: Is `IBM1848I` close enough to this, or do you think it's better to have a completely separate code (i.e. not one that's thrown by the compiler)?